cert-manager/cert-monitor.yaml

```diff
@@ -0,0 +1,94 @@
+apiVersion: v1
+data:
+  prometheus.yaml: |
+    global:
+      scrape_interval:     600s
+      evaluation_interval: 600s
+    rule_files:
+      - "cert_rules.yaml"
+    scrape_configs:
+      - job_name: certmanager
+        static_configs:
+        - targets: ['cert-manager.cert-manager:9402']
+        metric_relabel_configs:
+        - source_labels: ['namespace']
+          target_label: 'k8s_ns'
+  cert_rules.yaml: |
+    groups:
+    - name: certificate_rules
+      rules:
+      - record: certificate_expire_remaining_days 
+        expr: (certmanager_certificate_expiration_timestamp_seconds - time())/86400
+  sdexporter.yaml: |
+    static_metadata:
+    - metric: certificate_expire_remaining_days
+      type: gauge 
+      value_type: double 
+      help: Seconds until a certificate days
+kind: ConfigMap
```

Comment:
If humans are going to manage this, maybe move all these to the top of each block? :)

Reply:
what if those are humans that deal with lisp yamls, and read from the end to the beginning? :P

Kidding, corrected here!